### ISSUE TITLE
Regular term loss

### ISSUE THREAD

**Issue by XiangFeng66** [pranav-asthana/SplatSuRe#2]
Hello, excellent work !
From your experimental data, the regularization loss（only SRGS） seems to improve more over the baseline of the pre-trained 2D generative model (StableSR), which seems to exceed the 1dB PSNR. I don't know if my observation is correct.

**Comment by pranav-asthana** [comment]
Hi, thanks for your interest in our work!
Using SRGS improves quality over the pretrained SR model alone due to the addition of the low resolution image loss. Our method further improves the quality by limiting inconsistent SR supervision. Since our improvement is most significant in localized foreground regions (where LR is sufficient and SR hurts image quality), the whole image PSNR gain will be smaller and is less informative since it includes the entire image.
